docs/content/tutorial/saving-your-data.mdx

```diff
@@ -3,33 +3,45 @@ title: "Tutorial, part six: Saving your data | Dagster Docs"
 description: Learn how to use I/O managers to save your data.
 ---
 
-# Tutorial, part six: Saving your data
+# Tutorial, part six: Using Dagster to save and load your data
 
-You've written a data pipeline using assets that runs every hour. Currently, your assets are stored in a temporary folder that is deleted after closing the `dagster` process. **Note**: If you set up Dagster without the tutorial, you may have set a `DAGSTER_HOME` environment variable. If so, then your assets will be stored in that directory.
-
-You likely want to store your assets in places such as tables in your database or files in cloud storage. To do this, Dagster has I/O managers that enable reading and writing data to storage systems.
+You've written a data pipeline using assets that runs every hour. Currently, you are manually storing and loading your data using the `json` and `pandas` libraries at the beginning and end of each asset. Dagster provides I/O managers that will handle reading and writing data to storage for you. I/O managers allow you to reduce boilerplate reading and writing code and easily change where your data is stored.
```

Comment:
Instead of leading with "Dagster provides I/O managers that will handle reading and writing data to storage for you." I think the main way I would message the point of using IO managers is that they help you separate business logic from IO.

As you mention, this makes code easier to test and easier to understand, and allows you to plug in generic built-in IO managers for various storage systems.

Reply:
I would also lead with the general idea that IO managers are very powerful and beneficial for some situations, but are a poor fit for other situations. And if you find yourself getting tripped up trying to write your pipeline to use IO managers, often the easiest way to make progress is to not use them.